vhdbuilder/packer/install-dependencies.sh

```diff
@@ -381,23 +381,14 @@ done
 # kubelet and kubectl
 # need to cover previously supported version for VMAS scale up scenario
 K8S_VERSIONS="
-1.17.0
```

Comment:
not really needed for cluster/agentpool creation, kept for old vmas scale up optimization